src/main/java/io/strimzi/kafka/quotas/StorageChecker.java

```diff
@@ -77,13 +81,25 @@ public void run() {
         }
     }
 
+    @Deprecated(forRemoval = true, since = "0.3.0")
     long checkDiskUsage() {
+        return totalDiskUSage(gatherDiskUsage());
+    }
+
+    long totalDiskUSage(Map<String, Long> diskUsage) {
+        return diskUsage.values().stream().mapToLong(Long::longValue).sum();
+    }
+
+    Map<String, Long> gatherDiskUsage() {
         return logDirs.stream()
                 .filter(Files::exists)
                 .map(path -> apply(() -> Files.getFileStore(path)))
                 .distinct()
-                .mapToLong(store -> apply(() -> store.getTotalSpace() - store.getUsableSpace()))
-                .sum();
+                .map(store -> {
+                    final Long usedSpace = apply(() -> store.getTotalSpace() - store.getUsableSpace());
+                    return new AbstractMap.SimpleImmutableEntry<>(store.name(), usedSpace != null ? usedSpace : 0);
```

Comment:
Curiosity - how does FileStore.name() relate to the a name kube, if at all?

Reply:
I need to fire it up in K8s to check. The FileStore documentation is quite vague on what it really represents.